Vienna/Sources/Models/Article.m

```diff
@@ -370,7 +370,7 @@ -(NSString *)tagArticleEnclosureLink
  */
 -(NSString *)tagArticleEnclosureFilename
 {
-    return [self.enclosure.lastPathComponent stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
+    return [self.enclosure.lastPathComponent stringByAddingPercentEncodingWithAllowedCharacters:NSCharacterSet.URLPathAllowedCharacterSet];
```

Comment:
I guess you wanted to use stringByRemovingPercentEncoding and got confused…

Reply:
Good catch, will fix this.